### PR TITLE
Terminology and docs cleanup

### PR DESCRIPTION
**Pull request by uint-jamf**
* Aligned the blueprint vs template terminology with what we've discussed (please verify!)
* Trimmed some info from README.md to keep it snappier. Is that OK? The info can still be found in the User Guide.
* Added links to the badges.
* Prepared the thing for crates.io - hoping to publish once this is merged.

### CODE REVIEW

File: docs/src/usage.md [hunk 1]
```diff
@@ -5,7 +5,7 @@
 View the available commands with `rendr --help`:
 
     ❯ rendr --help
-    A project scaffolding tool
+    rendr: project scaffolding tool
```
Comment (thecodesmith) [inline docs/src/usage.md:8]: Can fix this to be how it was

File: src/bin/cli.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 name: rendr
-about: rendr: a project scaffolding tool
+about: "rendr: a project scaffolding tool"
```
Comment (thecodesmith) [inline src/bin/cli.yml:2]: Here too

File: docs/src/usage.md (hunk 1 above)
Comment (uint-jamf) [reply docs/src/usage.md]: Done...

File: src/bin/cli.yml (hunk 2 above)
Comment (uint-jamf) [reply src/bin/cli.yml]: ...and done!

**Pull request merged**